app/javascript/mastodon/features/home_timeline/index.js

```diff
@@ -153,7 +153,7 @@ class HomeTimeline extends React.PureComponent {
           scrollKey={`home_timeline-${columnId}`}
           onLoadMore={this.handleLoadMore}
           timelineId='home'
-          emptyMessage={<FormattedMessage id='empty_column.home' defaultMessage='Your home timeline is empty! Visit {public} or use search to get started and meet other users.' values={{ public: <Link to='/timelines/public'><FormattedMessage id='empty_column.home.public_timeline' defaultMessage='the public timeline' /></Link> }} />}
+          emptyMessage={<FormattedMessage id='empty_column.home' defaultMessage='Your home timeline is empty! Follow more people to fill it up. {suggestions}' values={{ public: <Link to='/start'><FormattedMessage id='empty_column.home.suggestions' defaultMessage='See some suggestions' /></Link> }} />}
```

Comment:
Changing the string and string interpolation will most likely cause issues. Btw, there's a mismatch between `public` and `suggestions` even here.

Reply:
Yes, it will, but it can't be a reason to never change it. Besides, unlike Rails, I don't think the app will crash if it tries to render a variable that is not given. It will simply show `{public}` in that place.

Reply:
Just checked, and the translation framework is smart enough to detect the translated string is outdated (from inspecting the mismatched passed/expected interpolation values) and decides to use the source string. So that's fine by me.